### PR TITLE
ADEN-13967 | Add DTCO inc1 tags

### PR DESCRIPTION
**Pull request by vforge**
Consume tags added in https://github.com/Wikia/unified-platform/pull/16899

### CODE REVIEW

File: @types/ads/index.d.ts [hunk 1]
```diff
@@ -32,6 +32,12 @@ interface AdsContext {
 	// context set only on MediaWiki
 	opts?: MediaWikiAdsOpts;
 	targeting?: MediaWikiAdsTargeting;
+	// The following code will be removed in INC2 of DTCO
```
Comment (adi-wesolowski-fandom) [inline @types/ads/index.d.ts:35]: such a comment is not needed, as long as there is a task in jira (where can be added sentence like "remove code made in 'inc1'" or smth like that).

**Review by adi-wesolowski-fandom — DISMISSED** [review]

File: src/platforms/shared/context/targeting/targeting-strategies/interfaces/common-targeting-params.ts [hunk 2]
```diff
@@ -10,4 +10,8 @@ export interface CommonTargetingParams {
 	s2?: string;
 	wpage?: string;
 	word_count?: string;
+	// The following code will be removed in INC2 of DTCO
+	main_vertical?: string;
```
Comment (jbienkowski-fandom) [inline src/platforms/shared/context/targeting/targeting-strategies/interfaces/common-targeting-params.ts:14]: we have to be careful not to hit max URI length limit for the browser nor HTTP server (usually ~2000 chars)
current URI from production:
```
echo 'https://beacon.wikia-services.com/__track/special/adengadinfo\?a\=499345\&ad_status\=requested\&advertiser_id\=\&beacon\=rFAVy-UnzJ\&bidder_0\=\&bidder_1\=0.20\&bidder_10\=\&bidder_11\=0.17\&bidder_12\=16fdm2o\&bidder_14\=\&bidder_17\=\&bidder_19\=\&bidder_2\=\&bidder_21\=\&bidder_25\=\&bidder_28\=\&bidder_30\=\&bidder_31\=\&bidder_32\=\&bidder_33\=\&bidder_34\=\&bidder_37\=\&bidder_38\=\&bidder_4\=\&bidder_8\=\&bidder_9\=\&bidder_won\=improvedigital\&bidder_won_price\=0.68\&browser\=OSX+Chrome+122\&c\=unknown\&cb\=48730\&ck\=unknown\&country\=PL\&creative_id\=\&device\=desktop\&is_uap\=0\&kv_ah\=36228\&kv_lang\=en\&kv_pos\=top_leaderboard\&kv_rv\=1\&kv_s0v\=movies\&kv_s1\=_starwars\&kv_s2\=wv-article\&kv_skin\=ucp_desktop\&labrador\=icPrebidOzone_B_97%3BicFeaturedVideoPlayer_A_90%3BicFeaturedVideoPlayer_A_80%3BicConnatixPlayer_A_75%3BicLABradorTest_A_100%3BicDeHeartbeatTracking_B_100\&lc\=unknown\&n\=-1\&opt_in\=yes\&opt_out_sale\=\&order_id\=\&pa_available\=1\&page_width\=1732\&ppid\=0271c699-c027-4f98-8dd6-380431c3dff8\&product_lineitem_id\=\&pv\=4\&pv_number\=4\&pv_number_global\=4\&pv_unique_id\=d14cbeb6-2350-4ed6-8f68-a29af29b7839\&rollout_tracking\=toi\&s\=ucp_desktop\&scroll_y\=0\&session_id\=ff68600f-aaf2-4bc6-84a1-e8941006bcf3\&short_page\=false\&slot_id\=6d02cb0d-275d-4b2a-821d-4b47805b4c96\&slot_size\=\&timestamp\=1711444314311\&topics_available\=1\&tz_offset\=-60\&u\=0\&ua\=Mozilla%2F5.0+%28Macintosh%3B+Intel+Mac+OS+X+10_15_7%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F122.0.0.0+Safari%2F537.36\&url\=https%3A%2F%2Fstarwars.fandom.com%2Fwiki%2FNew_Republic\&viewport_height\=914\&word_count\=17881\&x\=unknown' | wc -c
    1663
```

Comment (vforge) [reply src/platforms/shared/context/targeting/targeting-strategies/interfaces/common-targeting-params.ts:14]: @jbienkowski-fandom Trimmed in https://github.com/Wikia/ad-engine/pull/2419/commits/8a82d2df9e126f17df36d6439f45c9497eff5b35

File: @types/ads/index.d.ts (hunk 1 above)
Comment (vforge) [reply @types/ads/index.d.ts]: Fixed in https://github.com/Wikia/ad-engine/pull/2419/commits/8a82d2df9e126f17df36d6439f45c9497eff5b35

File: @types/ads/index.d.ts [hunk 3]
```diff
@@ -37,6 +37,11 @@ interface AdsContext {
 	// context set only on MediaWiki
 	opts?: MediaWikiAdsOpts;
 	targeting?: MediaWikiAdsTargeting;
+	// Cleanup in https://fandom.atlassian.net/browse/ADEN-13986
```
Comment (deriksolis) [inline @types/ads/index.d.ts:40]: have we created the new DTCO INCR 2 tickets? the cleanup comment will need updating

Comment (vforge) [reply @types/ads/index.d.ts:40]: @deriksolis I will keep the ticket number.

**Pull request merged**